lineman/app/app.coffee

```diff
@@ -15,6 +15,9 @@ angular.module('loomioApp', ['ngNewRouter',
   renderer = new marked.Renderer()
   renderer.link = (href, title, text) ->
     "<a href='" + href + "' title='" + (title || text) + "' target='_blank'>" + text + "</a>";
+  renderer.paragraph = (text) ->
+    text = text.replace(/([\s(\[]+|^)@([a-zA-Z0-9]+)([\s)\]\,\?]|$)/g, "$1<a class='lmo-user-mention' href='/u/$2'>@$2</a>$3")
```

Comment:
That regex is really great!
Pasting it into https://regex101.com/  is fun. Explains how it works really nicely.

It looks like you just need to add a `\.` in there..

```
([\s(\[]+|^)@([a-zA-Z0-9]+)([\s)\]\,\?\.]|$)
```

However I just looked at what we do on the rails side.. twitter-text library looks like it deserves some consideration.

https://github.com/twitter/twitter-text/tree/master/js

We already use it in the back end.. https://github.com/twitter/twitter-text